### PR TITLE
Styling feature

### PR DESCRIPTION
**Pull request by Armynerh**
Added JSX element to  project (without adding extra functionality)
 The final design matches Zeplin's design (this includes all elements + the same styling). However, I had to implement an extra field for categories and a button for removing books.
Didn't use the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/App.css
Comment (Strangeal) [inline src/App.css:1]: - [x] Great work on the styling of your project in this milestone. It's evident that you've put effort into creating an appealing visual presentation. However, I'd like to draw your attention to the project requirements regarding the design.

   -  Upon reviewing your `navbar`, I observed that it doesn't precisely match the design provided in the project requirements. Specifically, the profile `logo` and content at the far right side of the page is missing.

  - To align your project more closely with the design specifications, I recommend incorporating the `navbar` as specified in the project design.
  
<table>
<tr>
<th align=“middle”>main design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
Screenshot 2023-09-01 at 14 33 06
</td>
<td>
Screenshot 2023-09-01 at 14 32 00
</td>
</tr>
</table>

Happy coding! 😊🚀🍀

File: src/components/Addbook.js [hunk 1]
```diff
@@ -7,25 +7,33 @@ const AddBook = () => {
...
+  };
   return (
     <div className="flex formInput">
       <form className="auto">
         <input type="text" placeholder="Book Title." value={title} onChange={(event) => setTitle(event.target.value)} />
         <input type="text" placeholder="Author." value={author} onChange={(event) => setAuthor(event.target.value)} />
+        <input type="text" placeholder="category" value={category} onChange={(event) => setCategory(event.target.value)} />
         <button type="submit" onClick={handleAddbook}>ADD BOOK</button>
       </form>
     </div>
```
Comment (Strangeal) [inline src/components/Addbook.js]: - [x] Great job so far with the styling of your add form! 👏 While your progress looks good, there are a couple of suggestions to make it even better:

  - I recommend **removing** the `category` input field you added to your form. In the previous milestone, it was mentioned that you should swap the `category` input field in your design for `author`. This adjustment will ensure alignment with the project requirements.

  - It appears that your form is missing an `ADD NEW FORM` header, which is a part of the main design. To maintain consistency with the design, kindly consider adding this header to your form.
  
<table>
<tr>
<th align=“middle”>main design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
Screenshot 2023-09-01 at 14 35 46
</td>
<td>
Screenshot 2023-09-01 at 14 14 19
</td>
<td>

</td>
</tr>
</table>

Happy coding! 😊🚀🍀

File: src/App.css
Comment (Strangeal) [inline src/App.css:1]: - [x] Great job so far on styling your card component! 🚀 Your attention to detail is evident. However, I'd like to provide some feedback to help you align your design more closely with the project requirements. 

   - 🔍 Specifically, please take note that the background color of your `remove` card button differs from the main design. To ensure consistency, consider adjusting the button's background color to match the design.

   - 🌟 Additionally, pay attention to the spacing between your card's details and the card buttons. Make sure it aligns with the design provided in the project requirements.

<table>
<tr>
<th align=“middle”>main design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
Screenshot 2023-09-01 at 14 59 29
</td>
<td>
Screenshot 2023-09-01 at 15 00 13
</td>
</tr>
</table>
Happy coding! 😊🚀🍀

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @Armynerh, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Nice *UI* design ✅ 
- Pixel Perfect Design ✅ 
- Professional `README` template ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅ 
- Correct `.github/workflows` for linters ✅ 
- Followed *best-practices* ✅ 
- Correct **Semantics** used ✅ 
- Well structured project ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED   
 ## Hi  @Armynerh 
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Good Job :+1:
 
 Now you can merge this branch and move on 1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :


 
 ## Optional suggestions

- [x] N/A

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**